subgraphs/uniswap-forks/templates/spookyswap.template.yaml

```diff
@@ -0,0 +1,87 @@
+specVersion: 0.0.2
```

Comment:
Did you create this templated folder and at the spookyswap template here? I am thinking this is not necessary since the template is in the configurations folder.